packages/sw-background-sync-queue/demo/sw.js

```diff
@@ -2,14 +2,14 @@ importScripts('../build/background-sync-queue.js')
 
 //initialize bdQ
 goog.backgroundSyncQueue.initialize({},{
-	onRetrySuccess: async (req, res)=>{
+	onRetrySuccess: async (hash, req, res)=>{
 		console.log("success", req, res);
 		var data = await res.json();
```

Comment:
In the rest of this source you're heavily using `let` and `const`. Is there a reason `var` is used here instead?